processor/routingprocessor/logs.go

```diff
@@ -105,7 +106,7 @@ func (p *logProcessor) route(ctx context.Context, l plog.Logs) error {
 		matchCount := len(p.router.routes)
 		for key, route := range p.router.routes {
 			_, isMatch, err := route.statement.Execute(ctx, ltx)
-			if err != nil {
+			if err != nil && p.config.ErrorMode == ottl.PropagateError {
```

Comment:
lgtm

Thanks for adding the error mode to the routing processor configuration! Can you please add a sentence in README and change log entry that would tell users if they use the `ignore` error mode then the payload will be routed to the default exporter in case of OTTL statements evaluation failure?

Reply:
@kovrus will do

Reply:
added